subgit/core.py

```diff
@@ -486,6 +489,108 @@ def pull(self, names):
                 log.info(f"Current commit summary on HEAD in git repo '{name}': ")
                 log.info(f"  {str(repo.head.commit.summary)}")
 
+    def delete(self, repo_names=None):
+        """
+        Method takes zero or one repo name as argument.
+
+        Checks if the repo(s) are valid git repo(s) and if there are any untracked changes.
+        """
+
+        config = self._get_config_file()
+
+        active_repos = self._get_active_repos(config)
+
+        active_repos_dict = config.get("repos", [])
+
+        if repo_names is None:  # If no names are specified in the command
```

Comment:
We try to avoid comments on the same line as it tends to make the line extremly wide very easily. Try to keep comments on the line before like
```suggestion
        # If no names are specified in the command
        if repo_names is None:
```